### PR TITLE
Added Add More Button Dynamically

### PR DESCRIPTION
**Pull request by abedinforhan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,13 +4,16 @@
   "description": "Example for image uploading in Node",
   "main": "app.js",
   "scripts": {
-    "start": "node app.js"
+    "start": "nodemon app.js"
```
Comment (tanvirstreame) [inline package.json:7]: "start": "node app.js",
"dev": "nodemon app.js"

File: package.json [hunk 2]
```diff
@@ -4,13 +4,16 @@
...
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
     "ejs": "^2.5.7",
     "express": "^4.16.2",
     "multer": "^1.3.0"
+  },
+  "devDependencies": {
```
Comment (tanvirstreame) [inline package.json:16]: You can install nodemon globally

File: public/script.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const fileField=document.getElementById('file-field')
+const addMoreBtn=document.getElementById('add-more')
+
+addMoreBtn.addEventListener('click',()=>{
+console.log('hi');
```
Comment (tanvirstreame) [inline public/script.js:5]: remove logs

File: public/script.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+const fileField=document.getElementById('file-field')
```
Comment (tanvirstreame) [inline public/script.js:1]: why you need this files

File: .gitignore [hunk 5]
```diff
@@ -1 +1 @@
-node_modules
+node_modules
```
Comment (tanvirstreame) [inline .gitignore:1]: add `public` here

File: upload.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
+const multer = require('multer');
+const path = require('path');
+const fs=require('fs')
```
Comment (tanvirstreame) [inline upload.js:3]: Do proper code format

File: upload.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+// Init Upload
+const upload = multer({
+    storage: storage,
+    limits:{fileSize: 100000000},
+    fileFilter: function(req, file, cb){
+      checkFileType(file, cb);
+    }
+  }).any()
+
```
Comment (tanvirstreame) [inline upload.js:23]: Dont use extra spaces, You can have 1 or 2 line only

**Comment by tanvirstreame** [comment]
Make good commit message `commit` should not be any message

**Review by asifgenius — APPROVED** [review]

**Pull request merged**